src/components/Cardlist/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react";
+import Card from "../Card";
+import styles from "./index.module.css";
+
+interface IProps {
+  data: {
+    body_text: string;
+    header_text: string;
+    id: number;
+  }[];
+}
+
+const CardList: FC<IProps> = ({ data }) => {
+  return (
+    <div className={styles.card_list}>
+      {data.map((e) => {
```

Comment:
it is better to give better names for variables, what is data exactly? what is e? better names would be cardsInfo, or just cards and eachCard respectively